packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -86,19 +95,24 @@ class Product extends Component {
                 <div className={classes.name}>{item.name}</div>
                 {options}
                 <div className={classes.quantity}>
-                    <select
-                        className={classes.quantitySelect}
-                        value={item.qty}
-                        readOnly
-                    >
-                        <option value={item.qty}>{item.qty}</option>
-                    </select>
-                    <span className={classes.quantityOperator}>{'×'}</span>
-                    <span className={classes.price}>
-                        <Price currencyCode={currencyCode} value={item.price} />
-                    </span>
+                    <div className={classes.quantityRow}>
+                        <select
```

Comment:
We do have a [Select](https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/src/components/Select/select.js) component if you need it.

Reply:
I actually didn't add/change the selector, just added a wrapper around it for formatting. If it's OK with you, I'd like to leave the Select component change to another PR.